### ISSUE TITLE
Path address simplification

### ISSUE THREAD

**Issue by ististudio** [BookStackApp/BookStack#1546]
When I create a page called "pagename" under the book called "bookname", the relative address becomes "./books/bookname/page/pagename". I'd like to simplify or customize this address like "./b/bookname/p/pagename". I hope that there will be new env items. Here is what I think:

* The default values are "books", "page", "chapter".
* If the values are set to "one", "two" "three", then it forms the relative address "./one/bookname/two/pagename" or "./one/bookname/three/chaptername".

Can this feature be implemented structurally? I had modified "/books/" from "app/Entities/Book.php" and "app/Entities/Page.php" to "/b/", but the book hasn't been redirected to "./b/bookname".

**Comment by ssddanbrown** [comment]
Thanks for your request @ististudio.

This is somewhat related to #1490 

If you could explain the benefits that this request would bring to BookStack users it would really help show what this request achieves.

**Comment by ististudio** [comment]
Thank you for your answer.

Does that mean I didn't delete the cache? I saw that the address to which the anchor tag pointed was changed, but the address of a document that was already generated did not change. So I saw "Page Not Found".

I wanted him/her to focus on the title of the document when I shared the address of a document with others. Especially when the title of a document is less than five characters, I found "/books/" and "/pages/" annoying. And to me, "/book's'/" and "/page/" seemed a little inconsistent. I wanted to customize, because your intentions would not be unnecessary.
